pkgs/top-level/all-packages.nix

```diff
@@ -7239,7 +7239,11 @@ with pkgs;
   cpcfs = callPackage ../tools/filesystems/cpcfs { };
 
   coreutils =  callPackage ../tools/misc/coreutils { };
-  coreutils-full = coreutils.override { minimal = false; };
+
+  # The coreutils above are built with dependencies from
+  # bootstrapping. We cannot override it here, because that pulls in
+  # openssl from the previous stage as well.
+  coreutils-full = (callPackage ../tools/misc/coreutils { }).override { minimal = false; };
```

Comment:
Looks good! Thank you! A minor nit: you can pass arguments right into callPackage:

```nix
coreutils-full = callPackage ../tools/misc/coreutils {
  minimal = false;
};
```